File: Python/Types of Matrix/Row matrix.py
# Row matrix is that has 1 row and multiple columns
A = [[1, 2, 3]]

if len(A) == 1 and len(A[0]) > 1:
    print("Row matrix: ");
    for row in range(0, len(A)):
        for col in range(0, len(A[row])):
            print("\t", A[row][col], end='')
        print()
else:
    print("=> This isn't a row matrix.")

File: C++/Types of Matrix/Row matrix.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
    // Row matrix is that has 1 row and multiple columns
    vector <vector<int>> A = {
        { 1, 2, 3 }
    };

    if (A.size() == 1 && A[0].size() > 1) {
        cout << "Row matrix: " << endl;
        for (int row = 0; row < A.size(); row++) {
            for (int col = 0; col < A[row].size(); col++) {
                cout << "\t" << A[row][col];
            }
            cout << endl;
        }
    } else {
        cout << "=> This isn't a row matrix.";
    }
}